### PR TITLE
Add a playbook to add hosts to SSH known_hosts

### PR DESCRIPTION
**Pull request by brtknr**
(cherry picked from commit ec731cc745b111b5c485f21930bcb34deb565418)

Conflicts:
	ansible/euclid.yml

### CODE REVIEW

File: ansible/openhpc.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ---
+# Add cluster nodes to SSH known hosts
+- import_playbook: ssh-known-hosts.yml
+  when: ssh_known_hosts_enable | default(false)
```
Comment (markgoddard) [inline ansible/openhpc.yml]: Any reason not to add this to other appliances? k8s, swarm, etc.

Comment (oneswig) [reply ansible/openhpc.yml]: It's a good playbook.  Can we incorporate this into the top-level playbook for pulling in the cluster_setup role?

Comment (markgoddard) [reply ansible/openhpc.yml]: Could do. The container-infra-configure.yml playbook doesn't include that so we'd need to add it there also.

**Comment by oneswig** [comment]
This may be preempted by this cluster-infra PR - https://github.com/stackhpc/ansible-role-cluster-infra/pull/21

**Comment by oneswig** [comment]
Can we abandon this PR now we have the same effect in the `stackhpc.cluster-infra` role?

**Comment by brtknr** [comment]
Agree

**Pull request closed without merging**